source/versions/history.rst

```diff
@@ -1,6 +1,24 @@
 Package History
 ===============
 
+4.3.1 (2020-06-17)
+------------------
+
+Major Features and Improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Custom Action provides the ability for external applications to integrate custom dialogue management logic with MindMeld applications
+
+- Question Answerer can leverage semantic embeddings (Elasticsearch 7 is required)
```

Comment:
```suggestion
- Question Answerer can now leverage deep-learning based semantic embeddings to produce more relevant answers to queries
```

Reply:
I dont think we need to specify ES 7 here.

Reply:
It is specific to ES7+ right?